todo_app/templates/index.html

```diff
@@ -12,13 +12,52 @@ <h1 class="display-4">To-Do App</h1>
       <h2>Items</h2>
       <ul class="list-group mb-4">
         <!-- List the todo items here -->
-        {%for items in items%}<!--Used flask notation to loop through the items list-->
-          <li>{{ items.id }} - {{ items.status }} - {{ items.title }}</li><!--Used flask notation to add items into the unordered list-->
+        {%for card in cards%}
+          {% if card.idList == doing %}
```

Comment:
Very minor point, but it might be slightly nicer to keep as much logic in your Python code as possible. So when converting the JSON to Item objects, you could have done these checks on idList and set a card.status to nice value. Then index.html doesn't need to know about these idLists.